src/lib.rs

```diff
@@ -12,13 +12,15 @@ use bdk::database::{AnyDatabaseConfig, ConfigurableDatabase};
 use bdk::keys::bip39::{Language, Mnemonic, WordCount};
 use bdk::keys::{DerivableKey, ExtendedKey, GeneratableKey, GeneratedKey};
 use bdk::miniscript::BareCtx;
+use bdk::wallet::tx_builder::ChangeSpendPolicy;
 use bdk::wallet::AddressIndex as BdkAddressIndex;
 use bdk::wallet::AddressInfo as BdkAddressInfo;
 use bdk::{
     BlockTime, Error, FeeRate, KeychainKind, SignOptions, SyncOptions as BdkSyncOptions,
     Wallet as BdkWallet,
 };
-use std::convert::{From, TryFrom};
+use std::collections::HashSet;
+use std::convert::{From, TryFrom, TryInto};
```

Comment:
I'm getting a warning that `TryInto` is an unused import.

Reply:
Feel free to push a fix for this if you see this in your timezone before tomorrow, otherwise I'll just push a quick fix myself and merge tomorrow morning.

Reply:
fixed :)